package.json

```diff
@@ -103,9 +103,7 @@
     "prettier": "2.7.1",
     "ps-list": "^8.1.0",
     "qunit": "^2.17.2",
-    "rollup": "^2.75.6",
-    "rollup-plugin-size-snapshot": "^0.12.0",
-    "rollup-plugin-terser": "^7.0.2",
+    "rollup": "^3.9.0",
     "rollup-plugin-ts": "^3.0.2",
```

Comment:
what about the ts plugin?

Reply:
I saw you added and then removed it...
You must update typescript as well I believe

Reply:
Ohh I see that is the cause of the major fuck up in classes